test/package.mjs

```diff
@@ -32,6 +32,40 @@ const testSorted = (t, actualOrder, sourceName) => {
 	}
 };
 
+function toSentenceCase(originalString) {
+	let newString = originalString.charAt(0).toUpperCase() + originalString.slice(1); // Capitalize first letter.
+	if (newString.endsWith('.')) {
+		newString = newString.slice(0, -1); // Remove any ending period.
+	}
+
+	return newString;
+}
```

Comment:
Can you apply the same logic to this? 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/scripts/template/documentation.md.jst#L1

Maybe we can add another property here.

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/c10e2ba6aa700abe90d350f89275f314a8453d84/scripts/create-rule.mjs#L140

Reply:
Updated.